### ISSUE TITLE
[V3 Beta 2] Adding repo with non-default cog install location

### ISSUE THREAD

**Issue by TrustyJAID** [Cog-Creators/Red-DiscordBot#1094]
```
[11/11/2017 19:58] ERROR events on_command_error 159: Exception in command 'repo add'
Traceback (most recent call last):
  File "c:\program files\python36\lib\site-packages\discord\ext\commands\core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "C:\Program Files\Python36\Lib\site-packages\redbot\cogs\downloader\downloader.py", line 212, in _repo_add
    branch=branch
  File "C:\Program Files\Python36\Lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 541, in add_repo
    await self._save_repos()
  File "C:\Program Files\Python36\Lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 628, in _save_repos
    repo_json_info = {name: r.to_json() for name, r in self._repos.items()}
  File "C:\Program Files\Python36\Lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 628, in <dictcomp>
    repo_json_info = {name: r.to_json() for name, r in self._repos.items()}
  File "C:\Program Files\Python36\Lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 475, in to_json
    "folder_path": self.folder_path.relative_to(Path.cwd()).parts,
  File "c:\program files\python36\lib\pathlib.py", line 870, in relative_to
    .format(str(self), str(formatted)))
ValueError: 'E:\\github\\Testbot\\cogs\\RepoManager\\repos\\trusty' does not start with 'C:\\Users\\Trusty'
```

### Steps to reproduce
- Fresh install of Red V3
- Set cog path to something other than default on bot setup
- add repo with `[p]repo add name repo-link branch`

Error happens on Windows 10 with Python 3.6.2 Redbot Beta 2.

**Comment by tekulvw** [comment]
This issue actually stems from having the data location on an entirely different root path than the cwd. I'm planning on fixing this asap.

**Comment by tekulvw** [comment]
Fixed with #1150
